docs/docs/writing-hello-world.md

```diff
@@ -77,30 +79,34 @@ each of the different build types.
+* Release builds use a private key managed by an external signing process.
+
+2. Copy the `signing` directory in the `hello world`
+[sample](https://github.com/R3Conclave/conclave-tutorials/tree/HEAD/hello-world) repository into your project and
+update the paths in the enclave `build.gradle`. 
+
+The `signing` directory has the `privateKey` and `externalKey` signing types. See a description below:
 
-The `hello world` [sample](https://github.com/R3Conclave/conclave-tutorials/tree/HEAD/hello-world) repository contains some example
-keys that can be used with the `privateKey` and `externalKey` signing types. These can be found in `signing` directory.
+| Key Files              | Description                                                                                                                                                                   |
+|:-----------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| sample_private_key.pem | A 3072 bit RSA private key that can be used to test the ```privateKey``` signing type                                                                                         |
+| external_signing_*.pem | An AES encrypted 3072 bit RSA public/private key pair that can be used to test the ```externalKey``` signing type. The private key can be accessed with the password '12345'. |
 
-| Key Files              | Description                                                                                                                                                                  |
-|:-----------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| sample_private_key.pem | A 3072 bit RSA private key that can be used to test the ```privateKey``` signing type                                                                                        |
-| external_signing_*.pem | An AES encrypted 3072 bit RSA public/private key pair that can be used to test the ```externalKey``` signing type. The private key can be accessed with the password '12345' |
 
-Copy the `signing` directory from the SDK into your project and/or update the paths in the enclave `build.gradle`.
-Alternatively you can provide or [generate your own](signing.md#generating-keys-for-signing-an-enclave) keys.
+Alternatively, you can also [generate your own](signing.md#generating-keys-for-signing-an-enclave) keys.
 
-!!! important
-    These keys aren't whitelisted by Intel so you can't use them for real release builds.
-    Only use these sample keys for the tutorial. Don't use them for signing your own enclaves!
+!!!Important
+
+    Use these sample keys only for the tutorial. Do not use these keys to sign your own enclaves as Intel hasn't
+    whitelisted these sample keys.
```

Comment:
The whole concept of whitelisting signing keys is now dropped by Intel. Delete this "important" box. It's wrong also because we shouldn't be guiding developers to copy a publically visible sample private key!

Reply:
Deleted.